src/components/home/sections/RelatedCards.jsx

```diff
@@ -16,7 +16,10 @@ const RelatedCards = (props) => {
               This card has supporting text below as a natural lead-in to
               additional content.
             </Card.Text>
-            <a href="#!" className="readMoreCardBtn">
+            <a
```

Comment:
you can use ```<Link/>``` here, and can you add target blank that the page will open in another tap

Reply:
Actually it is your work and I only tried to fix the bugs we found but I can clearly do that!